portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiesSiteLevelUseCase.testcase

```diff
@@ -0,0 +1,508 @@
+
+			UserBar.signOut();
+
+			User.firstLoginUI(
+				password = "test",
+				specificURL = "http://www.able.com:8080",
+				userEmailAddress = "test@liferay.com");
+
+			CookiePreferenceHandlingAdmin.enablePreferenceHandlingWithCookieConsentInSiteSettings(
+				baseURL = "http://www.able.com:8080",
+				portlet = "Preference Handling",
+				siteURLKey = "site-name-1");
+		}
+
+		task ("When: End User first visits www.baker.com on the site1") {
+			UserBar.signOut();
+
+			User.firstLoginUI(
+				password = "test",
+				specificURL = "http://www.baker.com:8080",
+				userEmailAddress = "test@liferay.com");
+
+			CookieNavigator.gotoSiteSettings(
+				baseURL = "http://www.baker.com:8080",
+				portlet = "Preference Handling",
+				siteURLKey = "site-name-1");
+		}
+
+		task ("Then: Verify that the same Site1's site-level cookie configuration is present.") {
+			FormFields.viewCheckboxChecked(fieldName = "Enabled");
+
+			FormFields.viewCheckboxChecked(fieldName = "Explicit Cookie Consent Mode");
+		}
+	}
+
+	@description = "This is a use case for LPS-170904 TC-1. Verify no cookies are set in Site level and Explicit mode, when the user is a first-time visitor"
+	@priority = 5
+	test VerifyNoCookiesForFirstVisitInSiteLevelExplicitMode {
+		property test.name.skip.portal.instance = "CookiesSiteLevelUseCase#VerifyNoCookiesForFirstVisitInSiteLevelExplicitMode";
+
+		task ("Given: Site Administrator enabled the cookie preference handling in site level and the explicit cookie consent mode.") {
+			Site.openSiteSettingsAdmin(siteURLKey = "guest");
+
+			Site.addVirtualHostsURLCP(
+				pageVirtualHosts = "true",
+				pageVirtualHostURL = "www.able.com");
+
+			UserBar.signOut();
+
+			User.firstLoginUI(
+				password = "test",
+				specificURL = "http://www.able.com:8080",
+				userEmailAddress = "test@liferay.com");
+
+			CookiePreferenceHandlingAdmin.enablePreferenceHandlingWithCookieConsentInSiteSettings(
+				baseURL = "http://www.able.com:8080",
+				portlet = "Preference Handling",
+				siteURLKey = "guest");
+		}
+
+		task ("When: End User first visits the site") {
+			UserBar.signOut();
+
+			CookieHelper.deleteAllCookiesAndRefreshPage();
+
+			User.firstLoginUI(
+				password = "test",
+				specificURL = "http://www.able.com:8080",
+				userEmailAddress = "test@liferay.com");
+		}
+
+		task ("Then: Verify No cookies are set, not even the functional cookie: GUEST_LANGUAGE_ID verify cookie banner is present.") {
+			CookieBanner.verifyCookieBannerIsPresent();
+
+			CookieConsentPanel.verifyCookiesForCookieCategoriesAreNotPresent();
+
+			CookieHelper.checkSelectedCookieIsNotExist(cookieName = "GUEST_LANGUAGE_ID");
+		}
+	}
+
+	@description = "This is a use case for LPS-170904 TC-10. Verify Site 1 cookies don't appear when visiting Site 2"
+	@priority = 5
+	test VerifySite1CookiesDontAppearWhenVisitingSite2 {
+		property test.name.skip.portal.instance = "CookiesSiteLevelUseCase#VerifySite1CookiesDontAppearWhenVisitingSite2";
```

Comment:
Please ass HttpOnly disabler for this test case #2

Reply:
Done